.github/workflows/amis-workflow.yml

```diff
@@ -0,0 +1,157 @@
+            sed -i -e '/^stackDevOpsCloudstrapper: /s/:.*$/: publish-amis-to-marketplace-stack-devopscloustrapper/' ${{ env.VARS_DIR }}/defaults.yaml
+            sed -i -e '/^stackCloudstrapper: /s/:.*$/: publish-amis-to-marketplace-stack-cloustrapper/' ${{ env.VARS_DIR }}/defaults.yaml
+            sed -i -e '/^devOpsAmi: /s/:.*$/: cloudstrapper-'"$VERSION"'/' ${{ env.VARS_DIR }}/defaults.yaml
+            sed -i -e '/^buildUbuntuAmi: /s/:.*$/: ami-09e67e426f25ce0d7/' ${{ env.VARS_DIR }}/build.yaml
+            sed -i -e '/^buildAgwVersion: /s/:.*$/: '"$GIT_REF"'/' ${{ env.VARS_DIR }}/build.yaml
+            sed -i -e '/^buildAgwPackage: /s/:.*$/: '"$PACKAGE_VERSION"'/' ${{ env.VARS_DIR }}/build.yaml
+            sed -i -e '/^taggedVersion: /s/:.*$/: '"$VERSION"'/' ${{ env.VARS_DIR }}/build.yaml
+      - name: Generate Cloudstrapper AMI
+        timeout-minutes: 120
+        run: |
+            echo "DEFAULTS"
+            cat ${{ env.VARS_DIR }}/defaults.yaml
+            echo "BUILD"
+            cat ${{ env.VARS_DIR }}/build.yaml
+            echo "CLUSTER"
+            cat ${{ env.VARS_DIR }}/cluster.yaml
+            ansible-playbook ${{ env.WORK_DIR }}/devops-provision.yaml -e "dirLocalInventory=${{ env.VARS_DIR }}"
+            echo "Waiting one minute for the instance to boot up."
+            sleep 60
+            ansible-playbook ${{ env.WORK_DIR }}/devops-configure.yaml -e "devops=tag_Name_publishAmisToMarketplaceDevopsCloudstrapper" -e "dirLocalInventory=${{ env.VARS_DIR }}" -i ${{ env.VARS_DIR }}/common_instance_aws_ec2.yaml -u ubuntu --skip-tags usingGitSshKey,buildMagma,pubMagma,helm,pubHelm
+            ansible-playbook ${{ env.WORK_DIR }}/devops-init.yaml -e "dirLocalInventory=${{ env.VARS_DIR }}"
+      - name: Notify success to Slack
+        if: success() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.2.0
+        env:
+          SLACK_CHANNEL: ${{ secrets.SLACK_CHANNEL_ARTIFACTS }}
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_OSS }}
+          SLACK_TITLE: "*Cloudstrapper AWS AMI artifacts have been published*"
+          SLACK_MESSAGE: "${{ steps.commit.outputs.title}}"
+          SLACK_USERNAME: "Cloud workflow"
+          SLACK_ICON_EMOJI: ":heavy_check_mark:"
+          SLACK_COLOR: "#00FF00"
+          SLACK_FOOTER: ' '
+      - name: Propagate variables for AGW AMI build
+        run: |
+            sed -i -e '/^buildAgwAmiName: /s/:.*$/: agw-ami-'"$VERSION"'/' ${{ env.VARS_DIR }}/build.yaml
+            sed -i -e '/^buildGwTagName: /s/:.*$/: publishAmisToMarketplaceAgw/' ${{ env.VARS_DIR }}/build.yaml
+            sed -i -e '/^packageRepoHost: /s/:.*$/: '"${{ env.PACKAGE_REPO_HOST }}"'/' ${{ env.VARS_DIR }}/build.yaml
+            sed -i -e '/^awsAgwAmi: /s/:.*$/: ami-09e67e426f25ce0d7/' ${{ env.VARS_DIR }}/cluster.yaml
+            sed -i -e '/^buildAgwVersion: /s/:.*$/: '"refs\/heads\/${{ github.base_ref }}"'/' ${{ env.VARS_DIR }}/build.yaml
```

Comment:
Can you add a todo comment that is a workarround :)

Reply:
oh yes!